### DISCUSSION TITLE
Where to post our tutorial videos?

### DISCUSSION THREAD

**Discussion by avallecam** [epiverse-trace/epiverse-trace.github.io#143] [General]
From the action items collected during the Summit, there is one action on Building RSE and data science teams that involved the creation of video tutorials to make the transition to GitHub.

Currently, the Showcases are recorded on Zoom and uploaded to Panopto ([link to on showcase online](https://www.lshtm.ac.uk/newsevents/events/epiverse-trace-spring-2023-showcase)).

Tagging @lgbermeo who may already record learning videos in the project, and @jamesmbaazam and @joshwlambert who are going to record a video soon.

**Comment by avallecam** [comment]
Two options could be YouTube and Panopto. Some factors I found interesting to consider are content reach and content copyright.

Here are some references comparing platforms I'm willing to review and discuss with the ones interested: 
- https://search.informit.org/doi/abs/10.3316/informit.902351864062766
- https://journals.gre.ac.uk/index.php/compass/article/view/1231
- https://www.panopto.com/blog/comparing-the-top-online-video-platforms/ 
- https://www.panopto.com/blog/learning-and-development-when-not-to-use-youtube-or-vimeo-for-hosting-business-videos/
- https://www.panopto.com/blog/youtube-isnt-enough-for-education/

**Comment by avallecam** [comment]
Following on one comment by @jamesmbaazam today about having interaction with viewers on YouTube comments, we can focus the decision on platforms based on the features that the video developers want to have or need to have for the material they are preparing.

**Reply by jamesmbaazam** [reply]
I'm only familiar with YouTube and have generally had good experiences with it. Viewers can ask questions, which may be answered by the poster or other viewers.

**Comment by lgbermeo** [comment]
Hi! For the pilot version 0.0.1 of the epi-training kit we made the videos and tutorials on YouTube. Although being a pilot test, these are not openly available. The idea is that with the final version of the course as well as the rest of the material, the videos will be open for free review. If anyone wants to access and review the pilot videos you can contact me to give you access.
